player.go

```diff
@@ -299,3 +299,12 @@ func (p *Player) setMedia(m *Media) error {
 	C.libvlc_media_player_set_media(p.player, m.media)
 	return getError()
 }
+
+// SetXWindow sets the X window to play on
+func (p *Player) SetXWindow(XID uint32) error {
```

Comment:
Please consider calling the parameter `windowID` instead of `XID`.

Reply:
Done